apps/launchpad/app/components/survey-modal/survey-modal.tsx

```diff
@@ -12,7 +12,7 @@ import {
   DialogFooter,
   StepIndicator,
 } from '@0xintuition/1ui'
-import { useGetListDetailsQuery } from '@0xintuition/graphql'
+import { useGetAtomsQuery, useGetListDetailsQuery } from '@0xintuition/graphql'
```

Comment:
Is this being added to GraphQL package? 

If so let's note it with a ticket to update on main and cut a release so we can keep up to date -- I'm happy to do that